app/assets/stylesheets/admin/modals.css.scss

```diff
@@ -0,0 +1,216 @@
+.reveal-modal-bg {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: black;
+  z-index: 1004;
+  display: none;
+  left: 0;
+}
+
+.reveal-modal, dialog {
+  visibility: hidden;
+  display: none;
+  position: absolute;
+  z-index: 1005;
+  width: 100vw;
+  top: 0;
+  border-radius: 0.4em;
+  border: 0px none;
+  left: 0;
+  background-color: white;
+  padding: 1.25rem;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.4);
+  padding: 1.875rem;
+}
+
+@media only screen and (max-width: 40em) {
+  .reveal-modal, dialog {
+    min-height: 100vh;
+  }
+}
+
+.reveal-modal .column, dialog .column, .reveal-modal .columns, dialog .columns {
+  min-width: 0;
+}
+
+.reveal-modal > :first-child, dialog > :first-child {
+  margin-top: 0;
+}
+
+.reveal-modal > :last-child, dialog > :last-child {
+  margin-bottom: 0;
+}
+
+@media only screen and (min-width: 40.063em) {
+  .reveal-modal, dialog {
+    width: 80%;
+    max-width: 62.5rem;
+    left: 0;
+    right: 0;
+    margin: 0 auto;
+  }
+}
+
+@media only screen and (min-width: 40.063em) {
+  .reveal-modal, dialog {
+    top: 6.25rem;
+  }
+}
+
+.reveal-modal.radius, dialog.radius {
+  border-radius: 3px;
+}
+
+.reveal-modal.round, dialog.round {
+  border-radius: 1000px;
+}
+
+.reveal-modal.collapse, dialog.collapse {
+  padding: 0;
+}
+
+@media only screen and (min-width: 40.063em) {
+  .reveal-modal.tiny, dialog.tiny {
+    width: 30%;
+    max-width: 62.5rem;
+    left: 0;
+    right: 0;
+    margin: 0 auto;
+  }
+}
+
+@media only screen and (min-width: 40.063em) {
+  .reveal-modal.small, dialog.small {
+    width: 40%;
+    max-width: 62.5rem;
+    left: 0;
+    right: 0;
+    margin: 0 auto;
+  }
+}
+
+@media only screen and (min-width: 40.063em) {
```

Comment:
Isn't there a way to avoid duplicating this line? can't each of the `.reveal-modal` blocks be nested in here?